main.tf

```diff
@@ -17,6 +17,7 @@ module "main" {
   version = "23.1.0"
 
   cluster_resource_labels    = var.tags
+  create_service_account     = var.gcp_service_account != "" ? false : true
```

Comment:
```suggestion
  create_service_account     = var.gcp_service_account == "" ? true :false
```

Reply:
I find the way it is currently written harder to parse.

Reply:
Agree, I'll fix this.